bench-tps/src/cli.rs

```diff
@@ -353,6 +361,39 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(|arg| is_within_range(arg, 1..))
                 .help("The number of unique destination accounts per transactions 'chunk'. Lower values will result in more transaction conflicts.")
         )
+        .arg(
+            Arg::with_name("bind_address")
+                .long("bind-address")
+                .value_name("HOST")
+                .takes_value(true)
+                .validator(solana_net_utils::is_host)
+                .help("IP address to use with connection cache"),
+        )
+        .arg(
+            Arg::with_name("client_node_id")
+                .long("client-node-id")
+                .value_name("PATH")
+                .takes_value(true)
+                .help("File containing a node id (keypair) to communicate with validators"),
+        )
+        .arg(
+            Arg::with_name("client_node_stake")
+                .long("client-node-stake")
+                .value_name("LAMPORTS")
+                .takes_value(true)
+                .help(
+                    "Stake of the client_node_id",
+                ),
+        )
+        .arg(
+            Arg::with_name("client_node_total_stake")
+                .long("client-node-total-stake")
+                .value_name("LAMPORTS")
+                .takes_value(true)
+                .help(
+                    "Total stake of the client_node_id",
```

Comment:
Can you edit these help texts to explain the difference between these two args?

Reply:
Why do we need these two, can we obtain from the network?

Reply:
I wanted to add it as follow up PR, but can add here, maybe even simpler.

My general concern is that I think that we don't need to set up CC parameters on client side. Because there is one way quic connections and client doesn't receive much so receive window should not really matter. I don't observe effect of setting up these parameter on the client in my limited experiments (haven't dig into that much)